packages/graphql-codegen-cli/tests/cli-flags.spec.ts

```diff
@@ -151,4 +151,17 @@ describe.only('CLI Flags', () => {
     const config = createConfig(args);
     expect(config.overwrite).toBeTruthy();
   });
+
+  it('Should interpolate environmental variables in YML', () => {
+    process.env['SCHEMA_PATH'] = 'schema-env.graphql';
+    mockConfig(`
+        schema: \${SCHEMA_PATH}
```

Comment:
@ardatan why not just `$SOME_VAR`?

Reply:
In the issue, the interpolated string has `{}.
`${API_ENDPOINT}`

Reply:
I know, but I think `$API_ENDPOINT` is better API (similar to usage in terminal), and shouldn't conflict with other syntax.

Reply:
But I think the standard is the other one.